app/src/main/java/com/kickstarter/features/pledgedprojectsoverview/ui/PledgedProjectsOverviewScreen.kt

```diff
@@ -58,14 +62,21 @@ fun PledgedProjectsOverviewScreen(
     modifier: Modifier,
     onBackPressed: () -> Unit,
     lazyColumnListState: LazyListState,
+    errorSnackBarHostState: SnackbarHostState,
     ppoCards: LazyPagingItems<PPOCardDataMock>,
-    totalAlerts: Int = 0
+    totalAlerts: Int = 0,
+    onSendMessageClick: (projectName: String) -> Unit
 ) {
     Box(
         modifier = Modifier.fillMaxSize(),
         contentAlignment = Alignment.Center
     ) {
         Scaffold(
+            snackbarHost = {
+                SnackbarHost(
+                    hostState = errorSnackBarHostState
```

Comment:
Hey! If this is indeed supposed to be an error snackbar should we use the design system's KSErrorRoundedText in KSAlerts.kt?

Reply:
Good callout, but I opted not to for now since we can only attach one snackbar to the screen in this way, and we may need to use this snackbar for more than one type of non-error message in subsequent work. I figured I'd spend time figuring out how to show more than one type of snackbar in a subsequent PR, and i should have renamed the this to be more generic instead of errorSnackbarHostState

Reply:
Ah, well you see that is the issue I'm running into right now 😅 since my Confirm Address flow also involves a snackbar